pkg/cli/cli.go

```diff
@@ -326,9 +326,10 @@ func PromptSelectionFromMap[K comparable, T any](c *command, choices map[K]T, to
 
 func PromptYesOrNo(out *os.File, in *os.File, text string) (bool, error) {
 	fmt.Fprint(out, text+" (y/N): ")
```

Comment:
`(Y/n)`

Reply:
Done. Also change the logic to accept lower and upper case values for both: `y` and `n`.